rita_client/src/exit_manager/exit_loop.rs

```diff
@@ -299,3 +304,24 @@ pub fn start_exit_manager_loop() {
         }
     });
 }
+
+/// Hardcode exit althea address, this is needed for transitioning routers
+fn update_exit_server_to_althea_addresses() {
+    let rita_client = settings::get_rita_client();
+    let mut exits = rita_client.exit_client.exits;
+    for (s, e) in exits.iter_mut() {
+        match &s[..] {
+            "test" => {
+                if e.althea_address.is_none() {
+                    {}
+                }
+            }
+            "us_west" => {
+                if e.althea_address.is_none() {
+                    {}
+                }
+            }
+            _ => {}
+        }
+    }
+}
```

Comment:
Need to update this and update althea firmware with althea_addresses once exits generate them